### DISCUSSION TITLE
Getting product info from WindowsTerminal.exe?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#15921] [General]
My shell has a built-in variable function, @verinfo, which works like this.

```
v:\> echo %@verinfo[C:\Windows\SystemApps\Microsoft.Windows.Search_cw5n1h2txyewy\SearchApp.exe,ProductVersion]
10.0.19041.3271
```

All its options (ProductVersion, FileVersion, et c.) fail (it returns an empty string) when used on WindowsTerminal.exe or wt.exe.  Any idea why?  In Explorer, "Properties" shows all the information.  Thanks.

**Comment by lhecker** [comment]
This is related to #15927, right? In that case we can continue to discuss it there. 🙂

**Comment by vefatica** [comment]
Yes, essentially the same issue.  I was intending to mention that here but you beat me to it.
